scripts/ios/build-sargon.sh

```diff
@@ -45,6 +45,18 @@ generate_ffi() {
     local TARGET_FOR_DYLIB_PATH="aarch64-apple-ios"
   fi 
   cargo run --features build-binary --bin sargon-bindgen generate --library target/$TARGET_FOR_DYLIB_PATH/release/lib$1.dylib --language swift --out-dir target/uniffi-xcframework-staging
+
+  if $release; then
+    echo "🚢 Ensuring Sargon build for release - that it will work for e.g. iOS wallet to archive."
+    sed -i '' 's/let useLocalFramework = false/let useLocalFramework = true/' Package.swift
+    swift build -c release || exit $?
+    echo "🚢 Swift Sargon builds for release ✅"
+
+    # Prepare for release
+    sed -i '' 's/let useLocalFramework = true/let useLocalFramework = false/' Package.swift
```

Comment:
And we ensure to set `let useLocalFramework = false` in the Package.swift BEFORE we copy it over! ensuring it is part of the release.